src/nim_status_client.nim

```diff
@@ -51,7 +51,7 @@ proc prepareLogging() =
       proc (logLevel: LogLevel, msg: LogOutputStr) {.gcsafe, raises: [Defect].} =
         try:
           if signalsManagerQObjPointer != nil:
-            signal_handler(signalsManagerQObjPointer, ($(%* {"type": "chronicles-log", "event": msg})).cstring, "receiveSignal")
+            signal_handler(signalsManagerQObjPointer, ($(%* {"type": "chronicles-log", "event": msg})).cstring, "receiveChroniclesLogEvent")
```

Comment:
What about usage of receiveSignal in setupRemoteSignalsHandling() ? Can stay?

Reply:
Yep, that should be fine.
The problem with this one is that we make recursive logging. 
`chronicles log output` -> `post a log event posted` -> `receiveSignal` -> `trace "..."` -> `chronicles log output` -> ...